plaidml2/bridge/keras/__init__.py

```diff
@@ -1061,7 +1061,7 @@ def not_equal(lhs, rhs):
 @_log_call
 def normalize_batch_in_training(x, gamma, beta, reduction_axes, epsilon=1e-3):
     x_shape = shape(x)
-    ndims = len(x_shape)
+    ndims = x.tensor.shape.ndims
```

Comment:
Just wondering what's going on here. It's unfortunate that this will cause another ffi round trip (on top of the one inside `shape(x)`).